<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="mainLay" stretch="0,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>400</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tabSettings">
         <attribute name="title">
          <string>Settings</string>
         </attribute>
         <widget class="QSlider" name="hSlider_K">
          <property name="geometry">
           <rect>
            <x>79</x>
            <y>40</y>
            <width>191</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>45</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QSlider" name="hSlider_B">
          <property name="geometry">
           <rect>
            <x>79</x>
            <y>90</y>
            <width>191</width>
            <height>20</height>
           </rect>
          </property>
          <property name="minimum">
           <number>-999</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QGroupBox" name="gbPointSet">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>160</y>
            <width>331</width>
            <height>191</height>
           </rect>
          </property>
          <property name="title">
           <string>GroupBox</string>
          </property>
          <widget class="QLineEdit" name="leX1">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>30</y>
             <width>71</width>
             <height>21</height>
            </rect>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="leY1">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>60</y>
             <width>71</width>
             <height>21</height>
            </rect>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="lblX1">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>21</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>X1</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblY1">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>21</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Y1</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="leY2">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>60</y>
             <width>71</width>
             <height>21</height>
            </rect>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="lblX2">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>30</y>
             <width>21</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>X2</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="leX2">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>30</y>
             <width>71</width>
             <height>21</height>
            </rect>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="lblY2">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>60</y>
             <width>21</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Y2</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnClear">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>30</y>
             <width>80</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnApply">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>60</y>
             <width>80</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Add line</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblInfo">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>311</width>
             <height>40</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="lblStatusInfo">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>150</y>
             <width>311</width>
             <height>30</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="lblStatus">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>101</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Status</string>
           </property>
          </widget>
         </widget>
         <widget class="QLabel" name="lbl_K">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>K, наклон</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl_B">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>B, сдвиг</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>120</y>
            <width>113</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox_K">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>40</y>
            <width>61</width>
            <height>22</height>
           </rect>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox_B">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>90</y>
            <width>61</width>
            <height>22</height>
           </rect>
          </property>
          <property name="minimum">
           <number>-999</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tabPointList">
         <attribute name="title">
          <string>Point List</string>
         </attribute>
         <widget class="QTableView" name="tableViewPList">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>11</y>
            <width>331</width>
            <height>341</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vLayVewArea"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>leX2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>leX1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>leY2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>leY1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSlider_K</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_K</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSlider_B</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_B</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_K</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hSlider_K</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_B</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hSlider_B</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
